README.md

```diff
@@ -67,89 +73,93 @@ After getting everything running, you can open the **Pleuston Frontend** applica
 
-### Parity Client Accounts
+Hostname   | External Port | Internal Url          | Local Url             | Description
+-----------|---------------|-----------------------|-----------------------|--------------
+`pleuston` | `3000`        | http://pleuston:3000  | http://localhost:3000 | [Pleuston](https://github.com/oceanprotocol/pleuston)
 
-If you run the `start_ocean.sh` script with the `--local-parity-node` option, you will have available a Parity Client instance with the following accounts enabled:
+You can reach it on http://localhost:3000
 
-Account | Password | Balance    
---------|----------|--------
-0x00bd138abd70e2f00903268f3db08f2d25677c9e | node0 | 10000000111000111000111000
-0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0 | secret | 100000000
-0xa99d43d86a0758d5632313b8fa3972b6088a21bb | secret | 100000000
+### Secret Store
 
+Controlled by the `--no-secret-store` config switch will start three containers:
 
-Use one of the above accounts to populate `PARITY_ADDRESS` and `PARITY_PASSWORD` in `brizo.env` file to avoid asccount `locked` issues from the keeper contracts.
+Hostname                    | External Ports   | Internal Url                          | Local Url              | Description
+----------------------------|------------------|---------------------------------------|------------------------|--------------
+`secret-store`              | `12000`, `32771` | http://secret-store:12000             | http://localhost:12000 | An instance of the Ocean Secret Store
+`secret-store-cors-proxy`   | `12001`          | http://secret-store-cors-proxy:12001  | http://localhost:12001 | A nginx proxy to enable CORS on the secret store
+`secret-store-signing-node` | `9545`           | http://secret-store-signing-node:9545 | http://localhost:9545  | A parity node to `sign` messages for the secret store and to `decrypt` and `encrypt`
 
-### Environment Variables
+### Spree Network
 
-The `start_ocean.sh` script and `.env` file sets defaults for the following environment variables but you can use these in combination with the Docker Compose files for further customization, e.g.:
+If you run the `start_ocean.sh` script with the `--local-spree-node` option, you will have available a keeper node instance with the following accounts enabled:
 
-```bash
-export REUSE_DATABASE="true"
-docker-compose --project-name=ocean -f docker-compose-no-pleuston.yml up
-```
+Account                                      | Password   | Balance    
+---------------------------------------------|------------|--------
+`0x00bd138abd70e2f00903268f3db08f2d25677c9e` | node0      | 10000000111000111000111000
+`0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0` | secret     | 100000000
+`0xa99d43d86a0758d5632313b8fa3972b6088a21bb` | secret     | 100000000
+
+Use one of the above accounts to populate `PARITY_ADDRESS` and `PARITY_PASSWORD` in `brizo.env` file to avoid asccount `locked` issues from the keeper contracts.
```

Comment:
```suggestion
Use one of the above accounts to populate `PARITY_ADDRESS` and `PARITY_PASSWORD` in `brizo.env` file to avoid account `locked` issues from the keeper contracts.
```

Reply:
changed thanks for finding.